sketchConverter.py

```diff
@@ -1,11 +1,17 @@
+from flask import Flask, render_template, request
+from PIL import Image
 import cv2
 import os
 import argparse
+import numpy
```

Comment:
it is a good idea to document on `README` what the prerequisites to run the program (command lines to install the packages).